src/index.js

```diff
@@ -62,11 +62,11 @@ export default class ReactImgix extends Component {
       height: node.scrollHeight,
       mounted: true
     })
-  }
+  };
```

Comment:
Might need to do the old upgrade... `^6.4.5` should match `^6.5` though, which is what I'm on personally